File: back-kotlin/src/main/kotlin/me/nicolas/stravastats/domain/business/FtpEstimate.kt
package me.nicolas.stravastats.domain.business

data class FtpEstimate(
    val available: Boolean = false,
    val ftp: Int = 0,
    val method: String = "",
    val methodLabel: String = "",
    val bestPower: Int = 0,
    val multiplier: Double = 0.0,
    val basedOnSeconds: Int = 0,
    val confidence: String = "unavailable",
    val source: String = "",
    val sourceKind: String = "none",
    val activityId: Long = 0,
    val activityName: String = "",
    val activityType: String = "",
    val activityDate: String = "",
    val windowDays: Int = 180,
    val activityCount: Int = 0,
)

File: back-go/internal/shared/domain/business/ftp_estimate.go
package business

type FtpEstimate struct {
	Available      bool    `json:"available"`
	Ftp            int     `json:"ftp"`
	Method         string  `json:"method"`
	MethodLabel    string  `json:"methodLabel"`
	BestPower      int     `json:"bestPower"`
	Multiplier     float64 `json:"multiplier"`
	BasedOnSeconds int     `json:"basedOnSeconds"`
	Confidence     string  `json:"confidence"`
	Source         string  `json:"source"`
	SourceKind     string  `json:"sourceKind"`
	ActivityID     int64   `json:"activityId"`
	ActivityName   string  `json:"activityName"`
	ActivityType   string  `json:"activityType"`
	ActivityDate   string  `json:"activityDate"`
	WindowDays     int     `json:"windowDays"`
	ActivityCount  int     `json:"activityCount"`
}
